semantics/executable-spec/small-steps.cabal

```diff
@@ -35,6 +37,8 @@ library
                      , free
                      , mtl
                      , transformers >= 0.5
+                     , QuickCheck
```

Comment:
We shouldn't need `QuickCheck` in the library. If there are tests, they should just be added to the test package.

Reply:
The tests were moved, and I forgot to remove this.